song-docs/source/introduction.rst

```diff
@@ -0,0 +1,156 @@
+==============
+Introduction
+==============
+
+
+What is SONG?
+======================
+
+SONG is a robust metadata and validation system used to quickly and reliably track genome metadata scattered across multiple cloud storage systems. 
+In the field of genomics and bioinformatics, metadata managed by simple solutions such as spreadsheets and text files require significant time and effort to maintain and ensure the data is reliable. 
+With several users and thousands of genomic files, tracking the state of metadata and their associations can become a nightmare. 
+The purpose of SONG is to minimize human intervention by imposing rules and structure to user uploads, which as a result produces high quality and reliable metadata with a minimal amount of effort.
+
+.. note::
+
+    **SONG** is a recurrsive acronym for **S**\ ONG's **O**\ ur **N**\ ew **G**\ NOS
+
+.. 
+    What SONG is NOT
+    ==================
+
+.. _introduction_features:
+
+Features
+======================
+
+- Synchronous and asynchronous metadata validation using `JsonSchema <http://json-schema.org>`_
+- Strictly enforced data relationships and fields
+- Optional **info** json fields for user specific metadata
+- Standard REST API that is easy to understand and work with
+- Simple and fast metadata searching
+- Export payloads for SONG mirroring
+- Clear and concise error handling
+- ACL security using OAuth2 and scopes based on study codes
+- Unifies metadata with object data stored in SCORE
+- Built-in `Swagger UI <https://song.cancercollaboratory.org/swagger-ui.html>`_ for API interaction
+
+.. _intro_data_sub_workflow_ref:
+
+Data Submission Workflow
+======================================
+
+The data submission workflow can be separated into 4 main stages:
+1. Metadata Upload (SONG)
+2. Metadata Saving (SONG)
+3. Object data Upload (SCORE)
+4. Publishing Metadata (SONG)
+
+
+The following diagram summarized the steps involved in successful data submission using SONG and SCORE:
+
+.. image:: song-workflow.svg
+
+
+.. _intro_projects_user_song_ref:
+
+Projects Using SONG
+======================
+
+.. generated at https://staticmapmaker.com/google/
+
+.. image:: https://maps.googleapis.com/maps/api/staticmap?autoscale=false&size=600x300&maptype=roadmap&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:1%7CToronto&markers=size:mid%7Ccolor:0xffb100%7Clabel:2%7CVirginia&markers=size:mid%7Ccolor:0x0a00ff%7Clabel:3%7CBerlin&markers=size:mid%7Ccolor:0x00d70b%7Clabel:4%7CHeidelberg
```

Comment:
The image does not show up on the preview (http://142.1.177.168:8100/introduction.html)